packages/sdk/src/client.ts

```diff
@@ -270,17 +277,26 @@ export function SoundClient({
 
     switch (mintSchedule.mintType) {
       case 'RangeEdition': {
-        return RangeEditionMinter__factory.connect(mintSchedule.minterAddress, signer).mint(
-          mintSchedule.editionAddress,
-          mintSchedule.mintId,
-          quantity,
-          affiliate,
-          txnOverrides,
-        )
+        const rangeMinter = RangeEditionMinter__factory.connect(mintSchedule.minterAddress, signer)
+        const mintArgs = [mintSchedule.editionAddress, mintSchedule.mintId, quantity, affiliate] as const
+
+        if (txnOverrides.gasLimit) {
+          return rangeMinter.mint(...mintArgs, txnOverrides)
+        }
+
+        try {
+          // Add a buffer to the gas estimate to account for node provider estimate variance
+          const gasEstimate = await rangeMinter.estimateGas.mint(...mintArgs, txnOverrides)
+
+          txnOverrides.gasLimit = scaleAmount({ amount: gasEstimate, multiplier: MINT_GAS_LIMIT_MULTIPLIER })
+        } catch (err) {}
```

Comment:
@PabloSzx @vigneshka lmk if you think I should do something with this error. Seems like we can just fail silently since we fall back to default gas estimation process